BREEZE.rst

```diff
@@ -0,0 +1,753 @@
+   Run tests in the environment   : ./.build/test_run [test-target] [nosetest options]
+   Run Docker compose command     : ./.build/dc [help/pull/...] [docker-compose options]
+
+Keeping images up-to-date
+=========================
+
+Updating dependencies
+---------------------
+
+If you change apt dependencies in the Dockerfile or change setup.py or
+add new apt dependencies or npm dependencies, you have two options how to update the dependencies.
+
+
+*
+  You can install dependencies inside the container using 'sudo apt install', 'pip install' or 'npm install'
+  (in airflow/www folder) respectively. This is useful if you want to test somthing quickly while in the
+  container. However, those changes are not persistent - they will disappear once you
+  exit the container (except npm dependencies in case your sources are mounted to the container). Therefore
+  if you want to persist a new dependency you have to follow with the second option.
+
+*
+  You can add the dependencies to the Dockerfile, setup.py or package.json and rebuild the image. This
+  should happen automatically if you modify any of setup.py, package.json or update Dockerfile itself.
+  After you exit the container and re-run `./breeze <./breeze>`_ the Breeze detects changes in dependencies,
+  ask you to confirm rebuilding of the image and proceed to rebuilding the image if you confirm (or skip it
+  if you won't confirm). After rebuilding is done, it will drop you to shell. You might also provide
+  ``--build-only`` flag to only rebuild images and not go into shell - it will then rebuild the image
+  and will not enter the shell.
+
+Note that during development, changing dependencies in apt-get closer to the top of the Dockerfile
+will invalidate cache for most of the image and it will take long time to rebuild the image by breeze.
+Therefore it is a recommended practice to add new dependencies closer to the bottom of
+Dockerfile during development (to get the new dependencies incrementally added) and only move them to the
+top when you are close to finalise the PR and merge the change. It's OK for development time to add separate
+``apt-get install`` commands similar to those that are already there (but remember to move newly added
+dependencies to the appropriate ``apt-get install`` command which is already in the Dockerfile.
+
+Pulling the images
+------------------
+
+Sometimes the image on DockerHub is rebuilt from the scratch. This happens for example when there is a
+security update of the python version that all the images are based on. In this case it is much faster to
+pull latest images rather than rebuild them from the scratch. Airflow Breeze will detect such case and
+will ask you to confirm to pull and build the image and if you answer OK, it will pull and build the image.
+You might also provide ``--force-pull-images`` flag to force pull latest images from DockerHub.
+
+Airflow Breeze flags
```

Comment:
I don't think we should list all the commands in the `.rst` file. The main reason I think that is because if we change a flag in the original script or modify more flags it is easy to forget to update that in the doc.

And a user can simply get this by `breeze --help` and will probably make this file small & concise :)